core/src/database/tx_sender.rs

```diff
@@ -831,6 +831,59 @@ impl Database {
         Ok(())
     }
 
+    #[cfg(test)]
+    pub async fn get_fee_payer_utxos_for_tx(
```

Comment:
i would rather these impls be inside the `test::common` module to not clutter the production code.

Reply:
yeah, you need to do it inside `impl Database { }` (you can add impls to the same struct anywhere inside a crate so this is no problem)